data/src/test/java/org/apache/iceberg/data/DeletesReadTest.java

```diff
@@ -240,17 +252,11 @@ public void testEqualityDeleteByNull() throws IOException {
     Assert.assertEquals("Table should contain expected rows", expected, actual);
   }
 
-  private static StructLikeSet rowSet(Table table) throws IOException {
-    return rowSet(table, "*");
+  private StructLikeSet rowSet(Table tbl) throws IOException {
+    return rowSet(tbl, "*");
```

Comment:
Why rename this variable?

Reply:
The `checkstyle` reports hidden variable issue because it has the same name between the parameter and class member. I updated the member `table` to `testTable` in the PR.  will rebase this accordingly.